openmc/opencg_compatible.py

```diff
@@ -118,6 +143,19 @@ def get_openmc_material(opencg_material):
 
 
 def is_opencg_surface_compatible(opencg_surface):
+    """Determine whether OpenCG surface is compatible with OpenMC geometry
+
```

Comment:
I would clarify this docstring to say that if a surface is "compatible" it means there is a one-to-one correspondence between surface types in OpenMC/OpenCG. There are some surfaces in OpenCG (*e.g.*, ``SquarePrism``) for which this method will return false, but which the ``opencg_compatible`` module can convert into multiple OpenMC ``Planes`` and ``Cells`` which combine to describe the same geometry.